docs/intro.rst

```diff
@@ -37,12 +39,19 @@ Key Features
 
 Design
```

Comment:
I think @ewencp mentioned this too. I think we can put this after the intro and config and before the operations